src/sentry/api/serializers/models/alert_rule_trigger.py

```diff
@@ -4,12 +4,27 @@
 
 import six
 
-from sentry.api.serializers import register, Serializer
-from sentry.incidents.models import AlertRuleTrigger, AlertRuleTriggerExclusion
+from sentry.api.serializers import register, serialize, Serializer
+from sentry.incidents.models import AlertRuleTrigger, AlertRuleTriggerAction, AlertRuleTriggerExclusion
 
 
 @register(AlertRuleTrigger)
 class AlertRuleTriggerSerializer(Serializer):
+    def get_attrs(self, item_list, user, **kwargs):
```

Comment:
I'd say these changes could be split into a separate pr. Let's just focus on the write path here.

Reply:
The tests actually rely on this. We use this serializer to get an object with triggers + actions (since using the model doesn't make this as easy).